Patch-Source: https://git.0pointer.net/mod_dnssd.git/commit/?id=be2fb9f6158f800685de7a1bc01c39b6cf1fa12c
---
From be2fb9f6158f800685de7a1bc01c39b6cf1fa12c Mon Sep 17 00:00:00 2001
From: Robby Workman <rworkman@slackware.com>
Date: Fri, 8 Feb 2013 14:14:08 -0600
Subject: mod_dnssd.c: Use ap_unixd_setup_child

This fixeds an undefined reference to unixd_setup_child in the
compiled module.  This looks to have changed in httpd-2.3.2 dev:
20090130.0 (2.3.2-dev)  Add ap_ prefix to unixd_setup_child().
---
 src/mod_dnssd.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/mod_dnssd.c b/src/mod_dnssd.c
index 6179276..3363701 100644
--- a/src/mod_dnssd.c
+++ b/src/mod_dnssd.c
@@ -24,6 +24,7 @@
 #include <ap_config.h>
 #include <apr_strings.h>
 #include <unixd.h>
+#include <mod_unixd.h>
 #include <apr_signal.h>
 #include <mpm_common.h>
 
@@ -576,7 +577,7 @@ static void child_process(apr_pool_t *p, server_rec *server, struct global_confi
 
     ap_assert(d);
 
-    unixd_setup_child();
+    ap_unixd_setup_child();
 
     if (pipe(sigterm_pipe_fds) < 0) {
         ap_log_error(APLOG_MARK, APLOG_ERR, 0, r.main_server, "pipe() failed: %s", strerror(errno));
-- 
cgit 

